cli/kaos_cli/facades/tests/test_backend_facade.py

```diff
@@ -0,0 +1,368 @@
+        # Arrange
+        self.arrange_test_fixtures(environment, backend, url, token, infra, kubeconfig, user)
+
+        test_inputs_active_contexts = [None, '', environment]
+
+        for env in test_inputs_active_contexts:
+            self.state_service.set(ACTIVE, environment=env)
+
+            # Act
+            active_context = self.facade.get_active_context()
+
+            # Assert
+            self.assertEqual(active_context, self.facade.state_service.get(ACTIVE, 'environment'))
+
+    @parameterized.expand(backend_facade_test_inputs)
+    def test_property_active_context(self, environment, backend, url, token, infra, kubeconfig, user):
+        # Arrange
+        self.arrange_test_fixtures(environment, backend, url, token, infra, kubeconfig, user)
+
+        # Act
+        active_context_property = self.facade.active_context
+
+        # Assert
+        self.assertEqual(active_context_property, environment)
+
+    @parameterized.expand(backend_facade_test_inputs)
+    def test_property_url(self, environment, backend, url, token, infra, kubeconfig, user):
+        # Arrange
+        self.arrange_test_fixtures(environment, backend, url, token, infra, kubeconfig, user)
+
+        # Act
+        url_property = self.facade.url
+
+        # Assert
+        self.assertEqual(url_property, url)
+
+    @parameterized.expand(backend_facade_test_inputs)
+    def test_property_user(self, environment, backend, url, token, infra, kubeconfig, user):
+        # Arrange
+        self.arrange_test_fixtures(environment, backend, url, token, infra, kubeconfig, user)
+
+        # Act
+        user_property = self.facade.user
+
+        # Assert
+        self.assertEqual(user_property, user)
+
+    @parameterized.expand(backend_facade_test_inputs)
+    def test_property_token(self, environment, backend, url, token, infra, kubeconfig, user):
+        # Arrange
+        self.arrange_test_fixtures(environment, backend, url, token, infra, kubeconfig, user)
+
+        # Act
+        token_property = self.facade.token
+
+        # Assert
+        self.assertEqual(token_property, token)
+
+    @parameterized.expand(backend_facade_test_inputs)
+    def test_property_kubeconfig(self, environment, backend, url, token, infra, kubeconfig, user):
+        # Arrange
+        self.arrange_test_fixtures(environment, backend, url, token, infra, kubeconfig, user)
+
+        # Act
+        kubeconfig_property = self.facade.kubeconfig
+
+        # Assert
+        self.assertEqual(kubeconfig_property, kubeconfig)
+
+    @parameterized.expand(backend_facade_test_inputs)
+    def test_method_init(self, environment, backend, url, token, infra, kubeconfig, user):
+        # Arrange
+        self.arrange_test_fixtures(environment, backend, url, token, infra, kubeconfig, user)
+
+        # Act
+        ## TODO: Rectify the following assertions after fixing the dir_build path for init() after merging PR #95
+        # self.facade.init(url, token)
```

Comment:
If this test is not really acting, then please mark it as skipped. Example from `pytest`: https://docs.pytest.org/en/latest/skipping.html#skipping-test-functions

Reply:
The unittest is made to skip now as suggested